tmva/tmva/src/DNN/Architectures/Cpu/Initialization.hxx

```diff
@@ -150,7 +150,7 @@ void TCpu<AFloat>::InitializeIdentity(TCpuMatrix<AFloat> & A)
 
 //______________________________________________________________________________
```

Comment:
If we change here, we should change for all other functions in the file

Reply:
I was looking here at https://lcgapp-services.cern.ch/root-jenkins/view/ROOT/job/root-makedoc-master/2590/consoleText and trying to touch as less as possible to make those warnings disappear.

Reply:
I think your changes are fine, I would just fully qualify all these `Matrix_t` etc. Also including the class name, so it's clear to the reader where these types are defined if they want to know more.

Reply:
Do you mean writing instead TCpuTensor<AFloat>::Matrix_t and TCpuTensor<AFloat>::Tensor_t?

Reply:
In this file only or in all the other occurences too?

Reply:
I mean only in the places that you changed in this PR. Sorry, I only noticed your comment now